pkg/discovery/probe/node_probe.go

```diff
@@ -196,11 +208,26 @@ func (nodeProbe *NodeProbe) createCommoditySold(node *api.Node, nodePodsMap map[
 		}
 	}
 
-	// Use Kubernetes service UID as the key for cluster commodity
+	// Create a new VMPMAccessCommodity to indicate if the node is schedulable. If the Node is schedulable, then
+	// the node will sell it. Otherwise, it doesn't sell. On the consumer side, if a pod can be scheduled by
```

Comment:
talked to Enlin. pods running on unschedulable nodes don't buy this VMPMAccessCommodity.